source/2019-05-12-the-ember-times-issue-98.md

```diff
@@ -58,12 +59,13 @@ Whether you want to come enjoy the breathtaking views, make new Ember friends, o
   
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [Use Empress to render the Ember RFCs 📃](https://github.com/emberjs/rfcs/pull/489)
```

Comment:
```suggestion
<!--alex ignore king-queen-->
## [Use Empress to render the Ember RFCs 📃](https://github.com/emberjs/rfcs/pull/489)
```

for [fixing the Travis build](https://travis-ci.org/ember-learn/ember-blog/builds/533817688?utm_source=github_status&utm_medium=notification)